drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPageTest.java

```diff
@@ -324,7 +338,7 @@ private void spyOperatorsDropdown() {
 
             return spy(new CEPOperatorsDropdown(operators,
                                                 editingCol));
-        }).when(page).newCepOperatorsDropdown(any());
+        }).when(page).newCepOperatorsDropdown(any(), any());
```

Comment:
You've reformatted the classes; so you may as well ensure your changes are correctly formatted too! ;-)